pkg/api/api.go

```diff
@@ -506,6 +506,26 @@ func (s *Service) contentLengthMetricMiddleware() func(h http.Handler) http.Hand
 	}
 }
 
+func (s *Service) downloadSpeedMetricMiddleware() func(h http.Handler) http.Handler {
```

Comment:
I think we could add a string argument to the method, which would serve as a label for metrics. This way, we can easily distinguish between 'bzz' and 'bytes' in the endpoint data, because the download speed depends on the logic behind the handlers, and the logic in 'bzz' is more complex than in 'bytes'.

Reply:
Very nice idea, thanks.